boards/catie/zest_core_nrf52832/zest_core_nrf52832-pinctrl.dtsi

```diff
@@ -50,4 +50,45 @@
 			low-power-enable;
 		};
 	};
+
+	pwm0_default: pwm0_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 7)>;
+		};
+	};
+
+	pwm0_sleep: pwm0_sleep {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 7)>;
+			low-power-enable;
+		};
+	};
+
+	pwm1_default: pwm1_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 8)>;
+		};
+	};
+
+	pwm1_sleep: pwm1_sleep {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 8)>;
+			low-power-enable;
+		};
+	};
+
+	pwm2_default: pwm2_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 9)>,
+					<NRF_PSEL(PWM_OUT1, 0, 10)>;
+		};
+	};
+
+	pwm2_sleep: pwm2_sleep {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 9)>,
+					<NRF_PSEL(PWM_OUT1, 0, 10)>;
```

Comment:
Same comment

Reply:
Resolved in [d485455](https://github.com/catie-aq/zephyr_zest-core-nrf52832/pull/31/commits/d485455ac891935d318bb0072f22ec1a495ab34b).